components/WeeklyPickButton/WeeklyPickButton.tsx

```diff
@@ -12,7 +12,14 @@ const WeeklyPickButton: React.FC<WeeklyPickButtonProps> = ({ team, src }) => {
     <div className="flex items-center">
       <RadioGroupItem value={team} id={team} />
       <Label htmlFor={team}>
-        <Image src={src} alt={team} width={48} height={48} priority />
+        <Image
+          className="weeklyPickImage"
+          src={src}
+          alt={team}
+          width={48}
+          height={48}
+          priority
+        />
```

Comment:
m: remove `weeklyPickImage` from className and replace with:

`max-h-12`

This translates to max height 48px using tailwind utility class.

Reply:
This is a good point. I'll fix this on the next push.